app.py

```diff
@@ -182,6 +182,25 @@ def login_():
     def logout_():
         return logout()
 
+    @app.route('/push-log', methods=['POST'])
+    def push_log():
+        if 'level' not in request.json or 'message' not in request.json:
+            raise ValueError('Request body missing one or more mandatory fields ["message", "level"]')
+
+        _json = request.json.get
+        level, message, extra = _json('level'), _json('message'), _json('extra')
+        if not (extra is None or type(extra) is dict):
+            raise ValueError('Extra param must be a valid json object')
+
+        logging_fun = getattr(logger, level.lower(), None)
+
+        if logging_fun is None:
+            raise ValueError('Invalid logging level requested')
+
+        logging_fun(message, extra=extra)
```

Comment:
issue: I believe the logging endpoint should set the `source` attribute. I don't think we'll have different sources hitting this very endpoint aside from our frontend, and in that case we can override the attribute

Long story short, I'd like logs coming from the FE to be marked with a specific source as part of the log structure (I think you can just add it to the `extra` dict, if I remember correctly @tmscarla )

Reply:
Done in new review, thanks!